specs/binding-kafka.spec/src/main/resources/META-INF/zilla/kafka.idl

```diff
@@ -251,12 +252,68 @@ scope kafka
             KafkaHeader[] headers;      // INIT + FIN (produce), INIT only (fetch)
         }
 
+        union KafkaMergedDataEx switch (uint8)
+        {
+            case 252: kafka::stream::KafkaMergedConsumerDataEx consumer;
+            case 1: kafka::stream::KafkaMergedFetchDataEx fetch;
+            case 0: kafka::stream::KafkaMergedProduceDataEx produce;
+        }
+
+        struct KafkaMergedProduceDataEx
+        {
+            int32 deferred = 0;         // INIT only (TODO: move to DATA frame)
+            int64 timestamp = 0;        // INIT only
+            int64 filters = -1;         // INIT only
+            KafkaOffset partition;      // INIT only
+            KafkaOffset[] progress;     // INIT only
+            KafkaKey key;               // INIT only
+            KafkaKey hashKey;           // INIT only
+            KafkaDelta delta;           // INIT + FIN
+            KafkaHeader[] headers;      // INIT + FIN (produce), INIT only (fetch)
+        }
+
+        struct KafkaMergedFetchDataEx
+        {
+            int32 deferred = 0;         // INIT only (TODO: move to DATA frame)
+            int64 timestamp = 0;        // INIT only
+            int64 filters = -1;         // INIT only
+            KafkaOffset partition;      // INIT only
+            KafkaOffset[] progress;     // INIT only
+            KafkaKey key;               // INIT only
+            KafkaKey hashKey;           // INIT only
+            KafkaDelta delta;           // INIT + FIN
+            KafkaHeader[] headers;      // INIT + FIN (produce), INIT only (fetch)
+        }
+
+        struct KafkaMergedFetchDataEx
+        {
+            int32 deferred = 0;         // INIT only (TODO: move to DATA frame)
+            int64 timestamp = 0;        // INIT only
+            int64 filters = -1;         // INIT only
+            KafkaOffset partition;      // INIT only
+            KafkaOffset[] progress;     // INIT only
+            KafkaKey key;               // INIT only
+            KafkaKey hashKey;           // INIT only
+            KafkaDelta delta;           // INIT + FIN
+            KafkaHeader[] headers;      // INIT + FIN (produce), INIT only (fetch)
+        }
+
+        struct KafkaMergedConsumerDataExKafkaMergedConsumerDataEx
+        {
+            KafkaOffset partition;
+        }
+
         union KafkaMergedFlushEx switch (uint8)
         {
-            case 252: kafka::stream::KafkaMergedConsumerFlushEx consumer;
+            case 8: kafka::stream::KafkaMergedOffsetCommitFlushEx offsetCommit;
```

Comment:
This should stay as KafkaMergedConsumerFlushEx for consistency with KafkaMergedConsumerDataEx, right?